base_import_async/models/base_import_async.py

```diff
@@ -30,7 +30,9 @@
 from openerp.models import fix_import_export_id_paths
 from openerp.tools.translate import _
 
-from openerp.addons.connector.queue.job import job, related_action
+from openerp.addons.connector.queue.job import (
```

Comment:
@pedrobaeza `connector` is an odoo dependency of this module.I can not find a case where this repository would be installed without the connector because this module is the only one into the repository.